genie-core/src/main/java/com/netflix/genie/core/properties/JobsUsersActiveLimitProperties.java

```diff
@@ -46,4 +48,15 @@
     private boolean enabled = DEFAULT_ENABLED;
     @Min(value = 1)
     private int count = DEFAULT_COUNT;
+    private Map<String, Integer> overrides = Maps.newHashMap();
+
+    /**
+     * Get the maximum number of jobs a user is allowed to run concurrently.
+     * Checks whether the user has a special limit associated, and if not it returns the global default.
+     * @param user the user name
```

Comment:
nitpick here: when I run the auto formatting in Intellij it will add a new line between the description and this first `@param`. To avoid these formatting subsequent commits can you fix it now? You might have done it outside of intellij?